daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -363,6 +363,23 @@ damlStartTests getDamlStart =
               withCurrentDirectory projDir $
                   callCommand $
                   unwords ["daml", "ledger", "allocate-party", "--port", show sandboxPort, "Bob"]
+        , testCase "run a daml deploy without project parties" $ do
```

Comment:
Can we add this to the other deployment tests in https://github.com/digital-asset/daml/blob/main/daml-assistant/daml-helper/test/DA/Daml/Helper/Test/Deployment.hs instead? I’m slightly hesitant to add more stuff to the integration tests.

Reply:
I tried to put it in the other location first, but `daml deploy` needs the DAML_PROJECT path etc. set, so in the deployment tests I would have to set up all environment variables. In the integration tests, we have the environment setup for free.

Reply:
Isn’t it only DAML_PROJECT that is needed? that doesn’t sound that bad.

Reply:
It also needs DAML_ASSISTANT and then wants to call it to build the package.

Reply:
Ah I see, that’s probably too annoying to set it up.

Reply:
That's what I was thinking.